alfons/fennel.moon

```diff
@@ -0,0 +1,49 @@
+--alfons.fennel
+-- Fennel compiler with syntax sugar macros
+
+fennel = require "fennel"
+{ :eval } = fennel
+compiler = require "fennel.compiler"
+specials = require "fennel.specials"
+{ "make-compiler-env": makeCompilerEnv } = specials
+{ "compile-string": compileString } = compiler
+utils = require "fennel.utils"
```

Comment:
These lines need to be aligned by the `=` sign. I would also appreciate if all requires came before the destructuring assignments.
```suggestion
fennel   = require "fennel"
compiler = require "fennel.compiler"
specials = require "fennel.specials"
utils    = require "fennel.utils"

{ :eval }                                = fennel
{ "make-compiler-env": makeCompilerEnv } = specials
{ "compile-string": compileString }      = compiler

```